genomics/4-Variation/fetch_sra_acc.sh

```diff
@@ -19,7 +19,7 @@ do
 done < current.txt
 
 # Split into Illumina and GridION datasets
-grep Illumina current_metadata.txt |cut -f15 > current_illumina.txt
+grep -e 'Illumina\|NextSeq' current_metadata.txt |cut -f15 > current_illumina.txt
 grep GridION current_metadata.txt |cut -f15 > current_gridion.txt
```

Comment:
What about the MinION datasets? Are they omitted intentionally?

Reply:
Merged to get the fixed metadata (this only makes things better), question is still valid.

Reply:
There's just so few (12) compared to the GridION datasets (248) that I didn't think it's worth getting them. Something to keep in mind for when we finished analysing the GridION datasets, or do you want them now too ?